src/Altinn.Notifications.Persistence/Repository/OrderRepository.cs

```diff
@@ -271,7 +271,10 @@ public async Task InsertStatusFeedForOrder(Guid orderId)
             if (await reader.ReadAsync())
             {
                 // Read order tracking (first row is always the order-level tracking)
-                var reference = await reader.GetFieldValueAsync<string>("reference");
+                string? reference = await reader.IsDBNullAsync(reader.GetOrdinal("reference"))
+                    ? null
+                    : await reader.GetFieldValueAsync<string>("reference");
+
```

Comment:
Would it be enough like this?
```suggestion
                var reference = await reader.GetFieldValueAsync<string?>("reference");
```
I gues either way should work.